pymcp/registry.py

```
@@ -36,3 +36,62 @@ def get_tools(self):
+                        text = f.read()
+                    return {
+                        "uri": uri,
+                        "mimeType": res["mimeType"],
+                        "text": text
+                    }
+                else:
+                    raise FileNotFoundError(f"Resource not found: {res.get('path', uri)}")
+        raise ValueError("Resource not found")
+
+    def register(self, *args, **kwargs):
+        return self.add_resource(*args, **kwargs)
```

Pylint (3.3.7)

[error] Instance of 'ResourceRegistry' has no 'add_resource' member

(E1101)